test/client/http_service.test.js

```diff
@@ -0,0 +1,265 @@
+const http = require('http');
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+
+import { buildClientTestHarness, extractTestFixtureDataPoints } from '../testutil';
+
+describe('With a mock server', () => {
```

Comment:
a test that handles the request when the test server is *not* running would be useful, too.